noxfile.py

```diff
@@ -160,6 +175,8 @@ def snippets(session):
         os.path.join('docs', 'snippets_orgs.py'),
         os.path.join('docs', 'snippets_findings.py'),
         os.path.join('docs', 'snippets_security_marks.py'),
+        os.path.join('docs', 'v1p1beta1', 'snippets_notification_configs.py'),
```

Comment:
Since the tests are in `snippets_test.py` you'll want to add that file here instead of these two files.